src/celrender/openclusterrenderer.h

```diff
@@ -27,7 +27,7 @@ class OpenClusterRenderer
 
     void add(const OpenCluster *cluster, const Eigen::Vector3f &offset, float brightness, float nearZ, float farZ) const;
 
-    void update(const Eigen::Quaternionf &viewerOrientation, float pixelSize, float fov) const;
+    void update(const Eigen::Quaternionf &viewerOrientation, float pixelSize, float fov, float zoom) const;
```

Comment:
it seems it makes sense to pass Observer instead of viewerOrientation and zoom

Reply:
i'm adding support for transform for camera to support VR, so viewerOrientation != observer.getOrientation(), the i intend to store camera transform in Render class.